sessions/006/README.md

```diff
@@ -551,3 +551,331 @@ pvc-0bd38196-2f5b-4a05-8917-b43bd2dde50b   20Gi       RWO            Delete
+	Topic: t1	Partition: 1	Leader: 1	Replicas: 0,2,1	Isr: 1,0,2
+	Topic: t1	Partition: 2	Leader: 1	Replicas: 2,1,0	Isr: 1,0,2
+	Topic: t1	Partition: 3	Leader: 1	Replicas: 1,2,0	Isr: 1,0,2
+	...
+	Topic: t1	Partition: 999	Leader: 1	Replicas: 0,1,2	Isr: 1,0,2
+```
+We can also get the topic t2's information even though it caused disk full when creating topic t2.
+If this command didn't show expected results, try to run the command later since zookeeper also needs time to do data recovery, 
+and Kafka needs time to complete the topic t2 creation because it failed to create before Zookeeper server disk full. 
+
+```
+$ kubectl-kafka bin/kafka-topics.sh --bootstrap-server $CLUSTER_NAME-kafka-bootstrap:9092 --describe --topic t2
+Topic: t2	TopicId: LCYz44eWSYmMA1xhY9BpdQ	PartitionCount: 1000	ReplicationFactor: 3	Configs: min.insync.replicas=2,message.format.version=3.0-IV1
+	Topic: t2	Partition: 0	Leader: 0	Replicas: 1,0,2	Isr: 0,1,2
+	Topic: t2	Partition: 1	Leader: 0	Replicas: 0,2,1	Isr: 0,1,2
+	Topic: t2	Partition: 2	Leader: 0	Replicas: 2,1,0	Isr: 0,1,2
+	Topic: t2	Partition: 3	Leader: 0	Replicas: 1,2,0	Isr: 0,1,2
+	...
+	Topic: t2	Partition: 999	Leader: 0	Replicas: 1,2,0	Isr: 0,1,2
+```
+
+Finally, we delete the old volumes to reclaim some space, and optionally set the retain policy back to Delete on new volumes.
+
+```sh
+$ kubectl delete pv $(kubectl get pv | grep Available | awk '{print $1}')
+persistentvolume "pvc-8f3ddfba-e73a-4469-92e1-c8abc62006cc" deleted
+persistentvolume "pvc-90f07fde-7482-4764-b013-a04aac4e41fb" deleted
+persistentvolume "pvc-f6b43572-30ff-4423-a2eb-50d6733c55ec" deleted
+
+$ kubectl patch pv $(kubectl get pv | grep "$CLUSTER_NAME-zookeeper" | awk '{print $1}') --type merge -p '
+    spec:
+      persistentVolumeReclaimPolicy: Delete'
+persistentvolume/pvc-62dc40e2-8732-4563-bd9c-97a480f35dbf patched
+persistentvolume/pvc-d33f37a9-de7e-4b6a-8ab1-e0522571ebb1 patched
+persistentvolume/pvc-e56566de-8b03-4505-8872-f12603dea51f patched
+
+$ kubectl get pv | grep $CLUSTER_NAME-zookeeper
+pvc-62dc40e2-8732-4563-bd9c-97a480f35dbf   100Mi      RWO            Delete           Bound    test/data-my-cluster-zookeeper-2                                     ocs-external-storagecluster-ceph-rbd            6m28s
+pvc-d33f37a9-de7e-4b6a-8ab1-e0522571ebb1   100Mi      RWO            Delete           Bound    test/data-my-cluster-zookeeper-1                                     ocs-external-storagecluster-ceph-rbd            6m30s
+pvc-e56566de-8b03-4505-8872-f12603dea51f   100Mi      RWO            Delete           Bound    test/data-my-cluster-zookeeper-0                                     ocs-external-storagecluster-ceph-rbd            6m31s
+```
+
+<br/>
+
+---
+### Example: online ZooKeeper recovery from corrupted storage
+
+This procedure has some tricky steps highlighted in bold, where you need to be extra careful to avoid losing data.
+**Don't use Minikube for this example, as it doesn't have full volume support.**
```

Comment:
```suggestion
This example has some tricky steps highlighted in bold, where you need to be extra careful to avoid losing data.
**Don't use Minikube, as it doesn't have full volume support.**
```